src/services/homework/hooks/index.js

```diff
@@ -4,6 +4,17 @@ const globalHooks = require('../../../hooks');
 const hooks = require('feathers-hooks');
 const auth = require('feathers-authentication');
 
+const filterApplicableHomework = hook => {
+	let uId = hook.params.account.userId;
+	hook.result.data = hook.result.data.filter(function(c){
+		return (new Date(c.availableDate).getTime() < Date.now()
+			&& c.courseId != null
+			&& c.courseId.userIds.indexOf(uId) != -1)
```

Comment:
Das funktioniert nur, wenn du im client ein $populate:['userIds'] mit anfügst, wenn das aber nicht passiert (z.B. in postman), gibt es ein `500: Cannot read property 'indexOf' of undefined` error. Glaube hier reicht dann einfach ein `(c.courseId.userIds || []).indexOf(uId)`